weave

```diff
@@ -680,9 +709,9 @@ with_container_fqdn() {
     COMMAND="$2"
     shift 2
 
-    CONTAINER_FQDN=$(docker inspect --format='{{.Config.Hostname}}.{{.Config.Domainname}}.' $CONTAINER 2>/dev/null) || return 0
+    CONTAINER_FQDN=$(docker inspect --format='{{.Config.Hostname}}.{{.Config.Domainname}}' $CONTAINER 2>/dev/null) || return 0
 
-    if echo "$CONTAINER_FQDN" | grep -qv '\.\.$' ; then
+    if echo "$CONTAINER_FQDN" | grep -Eq '[^.]+\..+' ; then
```

Comment:
Ah, the [commit](https://github.com/awh/weave/commit/4f6ace8b109ae9d9e91474dd95054e7fababa7fc) explains it. Nevermind.